### PR TITLE
Allow to navigate with tab on all handicap popup content

### PR DESCRIPTION
**Pull request by edellucien**
# How to

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [ ] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/popups/HandicapPopup/HandicapPopupElement.js [hunk 1]
```diff
@@ -103,18 +104,22 @@ const renderLinks = (intialText) => {
...
+    matched = true;
     replaced = replaceLinks(replaced, emailMatches, mailTo);
   }
   if (urlMatches.length) {
+    matched = true;
     replaced = replaceLinks(replaced, urlMatches, urlHref);
   }
-  return replaced;
+  return matched ? replaced : <span tabIndex={0}>{[intialText]}</span>;
 };
```
Comment (gevos) [inline src/popups/HandicapPopup/HandicapPopupElement.js:123]: This method looks weird. There is a lot of redundant code. Could you find an alternative?

File: src/popups/HandicapPopup/HandicapPopup.js [hunk 2]
```diff
@@ -52,8 +53,10 @@ function HandicapPopup({ feature }) {
 
   const equipmentStr = equipment.length ? (
     <div className="wkp-handicap-popup-element" key="Ausstattung">
-      <div className="wkp-handicap-popup-field-title">{t('Ausstattung')}</div>
-      <div className="wkp-handicap-popup-field-body">
+      <div className="wkp-handicap-popup-field-title" tabIndex={0}>
+        {t('Ausstattung')}
+      </div>
+      <div className="wkp-handicap-popup-field-body" tabIndex={0}>
```
Comment (gevos) [inline src/popups/HandicapPopup/HandicapPopup.js:59]: What about the aria role attribute for heading, link, etc.?

**Review by gevos — APPROVED** [review]
Some remarks.

Additionally, the popup header, e.g. "Olten" cannot be accessed via tab.

File: src/popups/HandicapPopup/HandicapPopupElement.js (hunk 1 above)
Comment (edellucien) [reply src/popups/HandicapPopup/HandicapPopupElement.js:123]: This method is not related to the current ticket. If you have a improvment idea I would be glad else I can't spend time on this feature sorry

File: src/popups/HandicapPopup/HandicapPopup.js (hunk 2 above)
Comment (edellucien) [reply src/popups/HandicapPopup/HandicapPopup.js:64]: aria role heading added to the titles

**Review by gevos — APPROVED** [review]
> Additionally, the popup header, e.g. "Olten" cannot be accessed via tab. :heavy_check_mark:

LGTM

**Pull request merged**